lib/internal/crypto/random.js

```diff
@@ -119,6 +120,76 @@ function randomFill(buf, offset, size, cb) {
   _randomBytes(buf, offset, size, wrap);
 }
 
+// Largest integer we can read from a buffer.
+// e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
+const RAND_MAX = 281474976710655;
+
+// Generates an integer in [min, max] range where both min and max are
+// inclusive.
+function randomInt(min, max, cb) {
+  // randomInt(max, cb)
+  // randomInt(max)
+  if (typeof max === 'function' || typeof max === 'undefined') {
+    cb = max;
+    max = min;
+    min = 0;
+  }
+  const isSync = typeof cb === 'undefined';
+  if (!isSync && typeof cb !== 'function') {
+    throw new ERR_INVALID_CALLBACK(cb);
+  }
+  if (!NumberIsSafeInteger(min)) {
+    throw new ERR_INVALID_ARG_TYPE('min', 'safe integer', min);
+  }
+  if (!NumberIsSafeInteger(max)) {
+    throw new ERR_INVALID_ARG_TYPE('max', 'safe integer', max);
+  }
+  if (!(max > min)) {
+    throw new ERR_OUT_OF_RANGE('max', `> ${min}`, max);
```

Comment:
```suggestion
  if (!(max >= min)) {
    throw new ERR_OUT_OF_RANGE('max', `>= ${min}`, max);
```

Since the ranges are inclusive now? It doesn’t make a lot of sense to explicitly specify two equal values at first sight, but the logical bound to require here is `max >= min`, not `max > min`